test/src/userobjects/SplitTester.C

```diff
@@ -0,0 +1,33 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "SplitTester.h"
+#include "MooseMesh.h"
+
+#include "libmesh/distributed_mesh.h"
+
+template <>
+InputParameters
+validParams<SplitTester>()
+{
+  return validParams<GeneralUserObject>();
+  // pars.set<MultiMooseEnum>("execute_on") = "timestep_begin";
+}
+
+SplitTester::SplitTester(const InputParameters & parameters) : GeneralUserObject(parameters) {}
+
+void
+SplitTester::execute()
+{
+  auto & m = _fe_problem.mesh().getMesh();
+  if (!dynamic_cast<DistributedMesh *>(&m))
+    mooseError("not using DistributedMesh but should be");
```

Comment:
For new use Caps on error messages in MOOSE. That's the way we do it today. We can debate about switching everything to lower case, but this is inconsistent.